core-services/pdc/src/main/java/org/silverpeas/core/pdc/pdc/service/DefaultPdcUtilizationService.java

```diff
@@ -32,7 +32,6 @@
 import java.util.List;
 
 import org.silverpeas.core.admin.component.ComponentInstanceDeletion;
-import org.silverpeas.core.index.search.model.AxisFilter;
 
 import org.silverpeas.core.util.CollectionUtil;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.silverpeas.core.util.CollectionUtil'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)